descriptors/ModuleDescriptor-template.json

```
@@ -11,21 +11,21 @@
             "POST"
           ],
           "pathPattern": "/search/index/indices",
-          "permissionsRequired": ["search.index.indices.item.post"]
+          "permissionsRequired": [ "search.index.indices.item.post" ]
         },
         {
           "methods": [
             "POST"
           ],
           "pathPattern": "/search/index/mappings",
-          "permissionsRequired": ["search.index.mappings.item.post"]
+          "permissionsRequired": [ "search.index.mappings.item.post" ]
         },
         {
           "methods": [
             "POST"
           ],
-          "pathPattern": "/search/index/resources",
-          "permissionsRequired": ["search.index.resources.collection.post"]
+          "pathPattern": "/search/index/records",
+          "permissionsRequired": [ "search.index.resources.collection.post" ]
```

```suggestion
          "permissionsRequired": [ "search.index.records.collection.post" ]
```

Let's keep permission name in sync with the endpoint name. Please, also rename it in the `permissionSets`.